cmake/tribits/ci_support/CDashQueryAnalyzeReport.py

```diff
@@ -1054,15 +1055,20 @@ def dateFromBuildStartTime(buildStartTime):
 #
 # Inputs:
 #
-#   testHistoryLOD [in]: List of test dicts for the same test.  This list nore
-#   its elements are modified in this call. (The base list object is shallow
-#   copied before it is sorted.)
+#   testHistoryLOD [in]: List of test dicts for the same test.  This input
```

Comment:
"Neither this input list nor its elements are modified..."

Reply:
Thanks @mhoemmen!  I fixed this in https://github.com/TriBITSPub/TriBITS/commit/af11d587ea4abfefbfc55635ce78b9cbda252f50.